### ISSUE TITLE
scikit-learn.metrics import custom metric function

### ISSUE THREAD

**Issue by leeprinxin** [keras-team/autokeras#1829]
How do I import scikit-learn.metrics into a custom metrics function to use, for example, the following example :

```python
def mean_tweedie_deviance(y_true, y_pred):
    return mean_tweedie_deviance(y_true, y_pred)

model.compile(loss='mean_squared_error', optimizer='adam', metrics=[mean_tweedie_deviance])
```

**Comment by haifeng-jin** [comment]
The example you give should work, right?
If you wang to set the `deviance` as the objective, just do `objective=keras_tuner.Objective('val_deviance', direction='min')`.
